codice/ddf-ui
# Update paging to be fuzzy

Updates the query feed to use fuzzy results

Testing:

1. Ingest some data
1. Search and note that the hits are fuzzy as well as the counts when you click the heartbeat icon
1. Verify that toggling the config disables the fuzzy results
![image](https://user-images.githubusercontent.com/22667297/101072084-b41c1480-355a-11eb-897a-6b71a988e3d3.png)

![image](https://user-images.githubusercontent.com/22667297/100920086-df89fb00-3497-11eb-9115-5d763a49c874.png)
![image](https://user-images.githubusercontent.com/22667297/100920229-0ba57c00-3498-11eb-89d4-bb168ab4b1d8.png)
